sdk/eventhubs/azure-messaging-eventhubs/src/main/java/com/azure/messaging/eventhubs/implementation/handler/CustomIOHandler.java

```diff
@@ -32,7 +32,13 @@ public void onTransportClosed(Event event) {
 
     @Override
     public void onUnhandled(Event event) {
-        // logger.verbose("Unhandled event: {}, {}", event.getEventType(), event.toString());
-        super.onUnhandled(event);
+        // logger.verbose("Unhandled event: {}, {}", event.getEventType(), event);
```

Comment:
You can uncomment this. Since it's verbose, it's most likely not logged and if verbose logging is enabled, then I guess logging this is okay.

Reply:
Yeah. It's very, very verbose when it is enabled, because there are so many reactor events we don't bother to listen to. I guess that's why it's DEBUG.